client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-location-dialog-settings.js

```diff
@@ -138,8 +138,7 @@ const ShippingZoneLocationDialogSettings = ( {
 					<FormTextInput value={ postcode || '' } onChange={ onPostcodeChange } />
 					<p>
 						{ translate(
-							'Postcodes containing wildcards (e.g. CB23*) ' +
-								'and fully numeric ranges (e.g. 90210...99000) are also supported.'
+							'Postcodes containing wildcards (e.g. CB23*) and fully numeric ranges (e.g. 90210…99000) are also supported.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Postcodes containing wildcards (e.g. CB23*) and fully numeric ranges (e.g. 90210...99000) are also supported.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).